### PR TITLE
AVATAR_SIZE environment variable

### PR DESCRIPTION
**Pull request by maxtacu**
# Proposed Changes
- Add AVATAR_SIZE variable that sets the size in px. Ex: AVATAR_SIZE: 500px
Default value when variable is not set is 128px.

## Checklist

- [x] Tested locally
- [x] Ran `yarn ci` to test my code
- [x] Did not add any unnecessary changes
- [x] All my changes appear after other changes in each file
- [ ] Included a screenshot (if adding a new button)
- [x] 🚀

### CODE REVIEW

File: src/components/Avatar/Avatar.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 
 import './Avatar.css';
 
 function Avatar(props) {
   const { src, srcSet, alt } = props;
+  const avatarSize = runtimeConfig.AVATAR_SIZE || null;
 
-  return (
+  return avatarSize ? (
+    <img
```
Comment (timothystewart6) [inline src/components/Avatar/Avatar.js:13]: What if we did something like


```js
    <img
      className={'avatar' + addShadow()}
      src={src}
      srcSet={srcSet}
      alt={alt}
      style={avatarSize ?{ width: avatarSize, height: avatarSize }: {}}
    />
```
That way we don't have to repeat the component and conditionally render it, we will just give it a style if it does?

**Review by timothystewart6 — COMMENTED** [review]
I have one suggestion otherwise it looks good!  Thank you!

File: src/components/Avatar/Avatar.js (hunk 1 above)
Comment (maxtacu) [reply src/components/Avatar/Avatar.js]: updated pr. Thanks for review.

**Comment by timothystewart6** [comment]
Looks like you need to update your snapshot.  You can do this by running `yarn test -u` and then committing those changes.

**Comment by timothystewart6** [comment]
Thanks for sticking with it!

**Pull request merged**